### DISCUSSION TITLE
Clarify the meaning of db.instance tag

### DISCUSSION THREAD

**Discussion by lujiajing1126** [apache/skywalking#9672] [General]
In the Java agent, `db.instance` tag is extensively used in many plugins,

- For JDBC: database name,
- For ElasticSearch: index name,
- For Cassandra: keyspace name,
- For HBase: Table name
- ...

But I found in the `redisson` which is a high level Redis SDK, `ip:port` is used for this tag. As I understand, database number should be used in all redis client cases, e.g. Jedis, Lettuce, Redisson, etc.

Also, the semantics of this tag is not clearly documented, https://github.com/apache/skywalking-java/blob/6fce184cfe95b3a4b7cf97421ee4efdb03d2c7cc/apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/tag/Tags.java#L53

**Comment by wu-sheng** [comment]
The cache has been changed, please take a check again. 

https://github.com/apache/skywalking/blob/master/oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/vservice/VirtualCacheProcessor.java

**Reply by lujiajing1126** [reply]
I didn't find `db.instance` usage in this case?

**Reply by wu-sheng** [reply]
No, there is no use in the analysis. It is just a tag.

**Reply by wu-sheng** [reply]
Sometimes, tags are added randomly by plugin, but not used in the analysis.

**Comment by wu-sheng** [comment]
Do you have any concerns about this?

**Reply by lujiajing1126** [reply]
Since different plugins are setting values with different meanings, like Redisson, I'm curious if we have agreement how we use this tag.

**Reply by wu-sheng** [reply]
db.instance actually should not be used in the cache relative plugins  actually. 

ip:port should already be set through peer.
In this particular case, you could remove it.
